Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.10.16-alpine3.21
+LABEL maintainer="gnonasis@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /backend
+
+COPY requirements.txt requirements.txt
+
+RUN apk add --no-cache gcc musl-dev libffi-dev \
+    && pip install --no-cache-dir -r requirements.txt\
+    && apk del gcc musl-dev libffi-dev

```

Comment:
It's generally a good practice to combine the installation and removal of build dependencies in a single `RUN` command to reduce the image size. However, since you are already doing this, ensure that all necessary runtime dependencies are not removed.